Makefile

```diff
@@ -208,7 +208,12 @@ ifeq ("$(ARCH)","arm64")
 	endif
 else ifeq ("$(ARCH)","arm")
 # ARM builds need to specify the correct C compiler
```

Comment:
This comment no longer makes sense. Move it down to the `CC=` line and update so it says `Cross compilation must specify the correct C compiler`.